### PR TITLE
Add `net.peer.ip` and `net.host.ip` to deprecated yaml

### PR DESCRIPTION
**Pull request by dyladan**
In otel JS we're working on updating our semantic conventions package. In order to make it non-breaking we are relying on the deprecated attributes to be present for the code generator. This adds `net.host.ip` and `net.peer.ip` to the deprecated model, removed in https://github.com/open-telemetry/opentelemetry-specification/pull/2614

### CODE REVIEW

File: model/registry/deprecated/network.yaml [hunk 1]
```diff
@@ -35,12 +35,24 @@ groups:
         stability: experimental
         brief: Deprecated, use `server.port` on client spans and `client.port` on server spans.
         examples: [8080]
+      - id: peer.ip
+        type: string
+        deprecated: "Replaced by `net.sock.peer.addr`."
+        stability: experimental
+        brief: Deprecated, use `net.sock.peer.addr`.
```
Comment (trask) [inline model/registry/deprecated/network.yaml:42]:
```suggestion
        deprecated: "Replaced by `network.peer.address`."
        stability: experimental
        brief: Deprecated, use `network.peer.address`.
```

File: model/registry/deprecated/network.yaml [hunk 2]
```diff
@@ -35,12 +35,24 @@ groups:
...
         type: string
         deprecated: "Replaced by `server.address`."
         stability: experimental
         brief: Deprecated, use `server.address`.
         examples: ['example.com']
+      - id: host.ip
+        type: string
+        deprecated: "Replaced by `net.sock.host.addr`."
+        stability: experimental
+        brief: Deprecated, use `net.sock.host.addr`.
```
Comment (trask) [inline model/registry/deprecated/network.yaml:54]:
```suggestion
        deprecated: "Replaced by `network.local.address`."
        stability: experimental
        brief: Deprecated, use `network.local.address`.
```

File: model/registry/deprecated/network.yaml (hunk 1 above)
Comment (dyladan) [reply model/registry/deprecated/network.yaml:42]: I was going off this and the PR it was deprecated in: https://github.com/open-telemetry/semantic-conventions/blob/main/schemas/1.25.0#L344

I don't see another entry in the schema for it?

File: model/registry/deprecated/network.yaml (hunk 2 above)
Comment (dyladan) [reply model/registry/deprecated/network.yaml:54]: same

File: model/registry/deprecated/network.yaml (hunk 1 above)
Comment (dyladan) [reply model/registry/deprecated/network.yaml]: Seems the mapping wasn't possible to do automatically. I see the attributes now.

File: .chloggen/deprecated-net.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: net
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add deprecated net attributes to registry
```
Comment (trask) [inline .chloggen/deprecated-net.yaml:13]:
```suggestion
note: Add previously deprecated net attributes to registry
```

**Review by lmolkova — APPROVED** [review]

**Pull request merged**